### PR TITLE
Use JDatabaseQuery for queries

### PR DESCRIPTION
**Pull request by wilsonge**
To be tested

### CODE REVIEW

File: plugins/installer/webinstaller/webinstaller.script.php [hunk 1]
```diff
@@ -21,8 +21,12 @@ class plginstallerwebinstallerInstallerScript
         public function postflight($route, $adapter)
         {
             $db = JFactory::getDbo();
-            $query = 'UPDATE ' . $db->quoteName('#__extensions') . ' SET ' . $db->quoteName('enabled') . ' = 1 WHERE ' . $db->quoteName('type') . ' = ' . $db->quote('plugin') . ' AND ' . $db->quoteName('element') . ' = ' . $db->quote('webinstaller');
+            $query = $db->getQuery(true);
+            $query->update($db->quoteName('#__extensions'))
+                ->set($db->quoteName('enabled') ' = 1')
```
Comment (zero-24) [inline plugins/installer/webinstaller/webinstaller.script.php]: @wilsonge we miss a `.` here. :smile: 

Correct would be:
`->set($db->quoteName('enabled') . ' = 1')`

**Comment by zero-24** [comment]
works (just tested with my own plugin)

File: plugins/installer/webinstaller/webinstaller.script.php (hunk 1 above)
Comment (wilsonge) [inline plugins/installer/webinstaller/webinstaller.script.php]: Oops :P

**Comment by beat** [comment]
Tested too, works :+1: 

Merging as we have 2 successful tests.

**Pull request merged**